src/pages/HomePage/views/PublicPoolView.vue

```diff
@@ -242,26 +275,72 @@ const barIndicators = computed<InfoBarType.Indicator[]>(() => [
   },
 ])
 
-const dashboardIndicators = computed<InfoDashboardType.Indicator[]>(() => [
-  {
-    iconName: ICON_NAMES.ethereum,
-    title: t('home-page.public-pool-view.user-deposit-title'),
-    value: userPoolData.value
-      ? `${formatEther(userPoolData.value.deposited)} stETH`
-      : '',
-  },
-  {
-    iconName: ICON_NAMES.arbitrum,
-    title: t('home-page.public-pool-view.available-to-claim-title'),
-    value: currentUserReward.value
-      ? `${formatEther(currentUserReward.value)} MOR`
-      : '',
-  },
-  {
-    title: t('home-page.public-pool-view.multiplier-title'),
-    value: `x${rewardsMultiplier.value}`,
+const dashboardIndicators = computed<InfoDashboardType.Indicator[]>(() => {
+  const indicators: InfoDashboardType.Indicator[] = [
+    {
+      iconName: ICON_NAMES.ethereum,
+      title: t('home-page.public-pool-view.user-deposit-title'),
+      value: userPoolData.value
+        ? `${formatEther(userPoolData.value.deposited)} ${
+            web3ProvidersStore.depositTokenSymbol
+          }`
+        : '',
+    },
+    {
+      iconName: ICON_NAMES.arbitrum,
+      title: t('home-page.public-pool-view.available-to-claim-title'),
+      value: currentUserReward.value
+        ? `${formatEther(currentUserReward.value)} ${
+            web3ProvidersStore.rewardsTokenSymbol
+          }`
+        : '',
+    },
+  ]
+
+  if (isChangeLockEnabled.value) {
+    indicators.push({
+      title: t('home-page.public-pool-view.multiplier-title'),
+      value: `x${rewardsMultiplier.value}`,
+    })
+  }
```

Comment:
Instead of creating one more variable and manually pushing you can do this
```ts
const dashboardIndicators = computed<InfoDashboardType.Indicator[]>(() => [
  {
    iconName: ICON_NAMES.ethereum,
    title: t('home-page.public-pool-view.user-deposit-title'),
    value: userPoolData.value
      ? `${formatEther(userPoolData.value.deposited)} ${
          web3ProvidersStore.depositTokenSymbol
        }`
      : '',
  },
  {
    iconName: ICON_NAMES.arbitrum,
    title: t('home-page.public-pool-view.available-to-claim-title'),
    value: currentUserReward.value
      ? `${formatEther(currentUserReward.value)} ${
          web3ProvidersStore.rewardsTokenSymbol
        }`
      : '',
  },
  ...(isChangeLockEnabled.value
    ? [
        {
          title: t('home-page.public-pool-view.multiplier-title'),
          value: `x${rewardsMultiplier.value}`,
        },
      ]
    : []),
])
```

Reply:
done